OCKSample/Supporting Files/Info.plist

```diff
@@ -22,25 +22,27 @@
 	<string>$(CURRENT_PROJECT_VERSION)</string>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
-    <key>NSAppTransportSecurity</key>
-    <dict>
-        <key>NSAllowsArbitraryLoads</key>
-        <false/>
-        <key>NSExceptionDomains</key>
-        <dict>
-            <key>localhost</key>
-            <dict>
-                <key>NSAllowsArbitraryLoads</key>
-                <true/>
-            </dict>
-        </dict>
-    </dict>
+	<key>NSAppTransportSecurity</key>
+	<dict>
+		<key>NSAllowsArbitraryLoads</key>
+		<false/>
+		<key>NSExceptionDomains</key>
+		<dict>
+			<key>localhost</key>
+			<dict>
+				<key>NSAllowsArbitraryLoads</key>
+				<true/>
+			</dict>
+		</dict>
+	</dict>
 	<key>NSHealthClinicalHealthRecordsShareUsageDescription</key>
 	<string>CareKit sample app would like to read clinical records from HealthKit</string>
 	<key>NSHealthShareUsageDescription</key>
 	<string>CareKit sample app would like to read from HealthKit</string>
 	<key>NSHealthUpdateUsageDescription</key>
 	<string>CareKit would like to write to your HealthKit data!</string>
+	<key>NSMotionUsageDescription</key>
```

Comment:
This is what was causing the crash after clicking `Device Motion`. Basically, the developer has to add a description for the request.

To do this, in Xcode click on `OCKSample/Supporting Files/Info` and then look for `Privacy - Health Update Usage Description` in the `Key` column. Hover over the row and tap the `+` that appears to add a new row. Begin typing `Privacy - Motion` and the row should autofill with the rest of the entry. In the `Value` column add `CareKit sample app would like to access your motion data` to this new row. The end result should look like the following:

<img width="907" alt="image" src="https://user-images.githubusercontent.com/8621344/163380273-245cb847-f725-4293-9b4b-57da55b4314e.png">